target/linux/ipq40xx/files-4.19/arch/arm/boot/dts/qcom-ipq4018-m018u0-s1.dts

```diff
@@ -0,0 +1,76 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "qcom-ipq4018-m018u0.dtsi"
+
+/ {
+	model = "Corewav-M018U0-S1";
```

Comment:
Why not "Corewav M018U0 SOM reference design S1"?

Reply:
Changed